balances/src/lib.rs

```diff
@@ -115,6 +115,15 @@ pub mod pallet {
 }
 
 impl<T: Config> Pallet<T> {
+	pub fn get_all_balances(
+		account: &T::AccountId,
+	) -> Vec<(CommunityIdentifier, BalanceEntry<T::BlockNumber>)> {
+		let mut balances: Vec<(CommunityIdentifier, BalanceEntry<T::BlockNumber>)> = vec![];
+		for community in <encointer_communities::Pallet<T>>::community_identifiers().iter() {
+			balances.push((*community, Balance::<T>::get(community, account.clone())));
+		}
+		return balances
+	}
```

Comment:
Minor efficiency tweak: you can remove the copy-dereferencing if you use `into_iter`.